### PR TITLE
Korean translation update

### PR DESCRIPTION
**Pull request by akintos**
mainly focused on the UI strings.

### CODE REVIEW

**Comment by lonqie** [comment]
hi there, thank you for contributing this - got a few notes/questions on a few things:

- anything with a numbered key will use a game translation, so you don't need to add the translation directly to the file unless you want it to be different (for Korean there will always be a game translation available)
- I like to keep game terminology consistent with how it is displayed in game, unless there is a good reason for changing it. noticed some strings that differ in your commit (i'll mark them in the line by line comments)

File: LocalizationStrings.json [hunk 1]
```diff
@@ -43,24 +49,28 @@
...
+                "Kr": "무장갑"
             },
             "Normal": {
                 "CodeKey": 2051696859,
                 "En": "Normal",
                 "Cn": "普通装甲",
                 "Th": "เกราะปกติ",
-                "Vi": "Thường"
+                "Vi": "Thường",
+                "Kr": "일반"
```
Comment (lonqie) [inline LocalizationStrings.json:73]: game translation: 일반장갑

**Comment by akintos** [comment]
in that case it would be better to remove those translations. I will work on it.

File: LocalizationStrings.json [hunk 2]
```diff
@@ -94,131 +104,156 @@
...
-                "Vi": "Valkyrie"
+                "Vi": "Valkyrie",
+                "Kr": "발키리"
             },
             "ETC": {
                 "CodeKey": 792628026,
                 "Cn": "其他",
-                "Vi": "Etc."
+                "Vi": "Etc.",
+                "Kr": "기타"
```
Comment (lonqie) [inline LocalizationStrings.json:168]: game translation: 그 외
this one I don't really mind being changed ("Etc." never made much sense to me in English)

File: LocalizationStrings.json [hunk 3]
```diff
@@ -94,131 +104,156 @@
...
-                "Vi": "Đường phố"
+                "Vi": "Đường phố",
+                "Kr": "시가"
             },
             "Outdoor": {
                 "CodeKey": 1063796482,
                 "Cn": "室外",
-                "Vi": "Ngoài trời"
+                "Vi": "Ngoài trời",
+                "Kr": "실외"
```
Comment (lonqie) [inline LocalizationStrings.json]: game translations:
Street: "시가지"
Outdoor: "야외"

**Comment by lonqie** [comment]
thanks, those ones highlighted are the only ones I had concerns about being different, and were not sure if the change had been done purposefully or just a result of manually entering them

**Pull request merged**